devenv/sync.py

```diff
@@ -155,7 +155,7 @@ def main(context: dict[str, str]) -> int:
         (
             (
                 "python migrations",
-                (f"{venv_dir}/bin/{repo}", "upgrade", "--noinput"),
```

Comment:
`sentry` doesn't actually have this problem because the postgres container itself creates the `sentry` db

Reply:
good to know - are you suggesting setting POSTGRES_DB to repo here?

regardless though, there are other databases that need to be initialized - which is why i changed this over to make

Reply:
it wouldn't work -- the container is shared by sentry and getsentry